views/lib/view.js

```diff
@@ -162,14 +187,17 @@ export default class View {
   /**
    * Converts an InflectionData, returned from an inflection tables library, into an HTML representation of an inflection table.
    * `messages` provides a translation for view's texts.
+   * @param {Object} options - Render options
```

Comment:
See above, can you update the description of the render method to reflect what it is currently doing?

Reply:
Sure, done!